actionpack/test/dispatch/server_timing_test.rb

```diff
@@ -100,15 +109,15 @@ def initialize(app)
 
       def call(env)
         status, headers, body = @app.call(env)
-        headers["Server-Timing"] = [headers["Server-Timing"], %(entry;desc="description")].compact.join(", ")
+        headers["server-timing"] = [headers["server-timing"], %(entry;desc="description")].compact.join(", ")
```

Comment:
```suggestion
        headers[ActionDispatch::Constants::SERVER_TIMING] = [headers[ActionDispatch::Constants::SERVER_TIMING], %(entry;desc="description")].compact.join(", ")
```

Same as above, since this is a middleware we have to use dynamically cased headers